### DISCUSSION TITLE
Property filter with useCollection hook

### DISCUSSION THREAD

**Discussion by joel1842** [cloudscape-design/components#173] [Q&A] [answered]
I am working with the property filter & using the useCollection hook. The issue I am having is not with filtering items by property but using free text with the property filter. When I search for something using free text, the items are highlighted under values on the search dropdown, but when I click enter the table is empty. I see that the free text works on the table demo, but I am unable to access the source code. Perhaps I am missing something in the documentation, but as far as I can tell I have everything set up correctly. If someone has run into this issue, or if someone can provide some guidance on where I may be going wrong please let me know!

**Comment by gethinwebster** [comment, marked as answer]
Could you share some of your source code here so that we can understand better what exactly you've tried so far?

**Reply by joel1842** [reply]
Hey there, I realized that any field that you want free text searching to work for has to have a contains (:) operator. This isn't fully clear in the documentation so I will leave this discussion here just in case anyone else runs into the same issue!
